BookingsApi/BookingsApi.Contract/Requests/SearchTermAndAccountTypeRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BookingsApi.Contract.Requests
+{
+    public class SearchTermAndAccountTypeRequest
+    {
+        public SearchTermAndAccountTypeRequest(string term)
```

Comment:
Could this have an additional(option) parameter for AccountType? So that it can all be handled here instead of elsewhere in the code.